src/Akka.Persistence.Sql.HelperLib/TagTableMigrator.cs

```diff
@@ -14,6 +14,7 @@
 using Akka.Persistence.Sql.Db;
 using Akka.Persistence.Sql.Journal.Types;
 using LinqToDB;
+using LinqToDB.Async;
```

Comment:
Things got moved, now this namespace is a thing for `ToListAsync`/`ToDictionaryAsync` etc.